### DISCUSSION TITLE
Добавить методу BelongsToMany - возможность вывода в линию

### DISCUSSION THREAD

**Discussion by asasem** [moonshine-software/moonshine#351] [Ideas]
(Прикручивал Spatie авторизацию)

При связи через 3 таблицу  В ресурс контроллере  BelongsToMany::make('Права доступа', 'permissions', new UserResource(),
                )->select() 

Таблица через меню выводиться как :
![image](https://github.com/moonshine-software/moonshine/assets/7801474/de27741a-aa80-4a5a-9504-61fc6c3c986c)

Можно ли сделать например метод inLine() вывести в линию... а не таблицей.?

Костыли я наваял в Трейте WithFields для себя сделал  ... но хотелось бы чтоб было официально        
        
```

public function fields(): array
    {
        return [
            Block::make(trans('moonshine::ui.resource.main_information'), [
                ID::make()
                    ->sortable()
                    ->showOnExport(),

                Text::make(trans('moonshine::ui.resource.role_name'), 'name')
                    ->required()
                    ->showOnExport(),


                BelongsToMany::make('Права доступа', 'permissions', new UserResource(),
                )->select()//*->inLine()*//


            ]),
        ];
    }
```
А стало бы вот так 
![image](https://github.com/moonshine-software/moonshine/assets/7801474/218d4988-e69a-4244-910e-e0836ac3008a)

**Comment by asasem** [comment]
Спасибо что реализовали оставил комментарий
 
Merge pull request [https://github.com/moonshine-software/moonshine/pull/358 from lee-to/belongs-to-many-inline](https://github.com/moonshine-software/moonshine/commit/5d88e0f2c13e254b3f4a1dc87f88ba5543a2c2dc)

**Reply by lee-to** [reply]
Пожалуйста! Стараюсь прислушиваться ко всем вашим идеям и просьбам
